saleor/dashboard/product/views.py

```diff
@@ -17,6 +18,7 @@
 
 
 @staff_member_required
+@permission_required('product.edit')
```

Comment:
Instead of copy-pasting the same set of decorators we should have one for each unique set of permissions. Something like `@product_manager_required`.

Reply:
This is a good idea when i implement Groups. I think.